data/presets/amenity/public_bookcase.json

```diff
@@ -3,23 +3,28 @@
     "fields": [
         "name",
         "public_bookcase/type",
+        "books",
         "operator",
         "capacity",
-        "lit"
+        "direction"
```

Comment:
The `direction` tag is not used very commonly on public bookcases. What's the intention behind offering the field by default for these features?

Reply:
So it can be more easily discovered.
`direction` in general is not often used, because it is not directly a object/access type.
However, it is often found on/near barrier and usually only accesible from one side.
It is recommended to add direction to such objects to prevent ambiguity with its position relative to other objects.
(like cameras, defibrillators, etc.)